### PR TITLE
Position help menu list on top

### PR DESCRIPTION
**Pull request by microbit-grace**
so that the beta warning bar doesn't obstruct it.

Private link to task: https://microbit-global.monday.com/boards/1550536443/pulses/1700869947

### CODE REVIEW

File: src/components/HelpMenu.tsx [hunk 1]
```diff
@@ -25,8 +25,9 @@ const HelpMenu = ({ ...rest }: HelpMenuProps) => {
   const feedbackDisclosure = useDisclosure();
   const intl = useIntl();
   const MenuButtonRef = useRef(null);
+  const ContainerRef = useRef(null);
   return (
-    <Box {...rest}>
+    <Box {...rest} ref={ContainerRef}>
```
Comment (microbit-matt-hillsdon) [inline src/components/HelpMenu.tsx:30]: Do we have 2 completely different solutions to this problem for the settings menu and the help menu?

Comment (microbit-grace) [reply src/components/HelpMenu.tsx:30]: Yes, I believe settings MenuList is using `zIndex = {2}`. Shall I change that to use portal instead?

Comment (microbit-matt-hillsdon) [reply src/components/HelpMenu.tsx:30]: Yeah I think that's reasonable - probably less fragile. Either way let's make them work the same way!

Comment (microbit-grace) [reply src/components/HelpMenu.tsx:30]: Change made here -->  [30b2bc3](https://github.com/microbit-foundation/ml-trainer/pull/488/commits/30b2bc34922e43fbb6b5c8571b785b9b24916c36)

File: src/components/HelpMenu.tsx [hunk 2]
```diff
@@ -25,8 +25,9 @@ const HelpMenu = ({ ...rest }: HelpMenuProps) => {
   const feedbackDisclosure = useDisclosure();
   const intl = useIntl();
   const MenuButtonRef = useRef(null);
+  const ContainerRef = useRef(null);
```
Comment (microbit-robert) [inline src/components/HelpMenu.tsx:28]: Can we lower case the ref names?

File: src/components/HelpMenuItems.tsx
Comment (microbit-grace) [inline src/components/HelpMenuItems.tsx:1]: Might as well lowercase these refs ones as well

File: src/components/HelpMenu.tsx (hunk 2 above)
Comment (microbit-grace) [reply src/components/HelpMenu.tsx]: [52ab443](https://github.com/microbit-foundation/ml-trainer/pull/488/commits/52ab44347e3f1543ea14c8c3d4e788cf0428fcfc)

**Review by microbit-robert — APPROVED** [review]
LGTM.

**Pull request merged**